syntax = "proto3";
package com.fiberhome.fums.proto;
option java_package = "com.fiberhome.fums.proto";   
option java_outer_classname = "CtrlApp";
import "CommonDefine.proto";

message CtrlAppData
{
	int64   uniqueId         = 1;
	string  appType          = 2;
	EnumDefineData.AppTypeEnum appTypeCode = 3;
	enum  CtrlTypeEnum
	{
		START      = 0;
		STOP       = 1;
		RESTART    = 2;
		HASTART    = 3;
		HARESTART  = 4;
	}
	CtrlTypeEnum   ctrlType   = 4;
}


message CtrlAppRstResponse
{	
	int64   uniqueId         = 1;
	CommonResponse  res      = 2;
}